apps/studio/components/interfaces/SQLEditor/UtilityPanel/UtilityTabResults.tsx

```diff
@@ -95,15 +95,15 @@ const UtilityTabResults = ({ id, isExecuting }: UtilityTabResultsProps) => {
               </p>
             </div>
           ) : (
-            <div>
+            <div className="max-w-[54rem]">
```

Comment:
Can we not just use the width of the container?

Reply:
Hi Jonny, I've made some tweaks and pushed.

Reply:
Hi @MildTomato - Just checking if you saw this one?